### ISSUE TITLE
Add Deterministic Behavior to Color Picking for Testing Purposes

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/mosaic#39]
The current *method* for *picking images/colors* is shown below:
https://github.com/lafemmephile/mosaic/blob/e1ec8c794a605e9a92b112e1f6f9a5084a5d870e/src/mosaic/image_sub_div.py#L155-L231

There are *many* calls to `random.choice` and a single call to `random.shuffle`. While this may add *uniqueness* to the final mosaic generated ... it makes the output of the mosaic generation **non-deterministic** (i.e. random). 

Implement a `flag` of sorts that switches off this *random* behavior for testing purposes.

**Comment by lafemmephile** [comment]
A private API like the following may work:
```python
def _shuffle():
    # thin wrapper around random.shuffle

def _choice():
    # thin wrapper around random.choice
```

**Comment by lafemmephile** [comment]
As it turns out, all that was needed was feeding a *seed* to the *random number generator*: https://docs.python.org/3/library/random.html#random.seed

**Comment by lafemmephile** [comment]
Implemented with: 9c4800bf50d3b10a6feff3fcdd2036e8f6b9e72a
